Owncloud iOs Client/AppDelegate.m

```diff
@@ -1285,10 +1294,14 @@ - (void) restoreDownloadsInProccessFromSystemWithIdentificator: (NSString*)ident
     
     NSURLSessionConfiguration *configuration = nil;
     
-    if (IS_IOS8 || IS_IOS9) {
-        configuration = [NSURLSessionConfiguration backgroundSessionConfigurationWithIdentifier:identifier];
+    if (k_is_sso_active || !k_is_background_active) {
+        configuration = [NSURLSessionConfiguration defaultSessionConfiguration];
     } else {
-        configuration = [NSURLSessionConfiguration backgroundSessionConfiguration:identifier];
+        if (IS_IOS8 || IS_IOS9) {
```

Comment:
Same here. Remove this if and use for all cases ``` backgroundSessionConfigurationWithIdentifier: ``` instead of ```backgroundSessionConfiguration``` that is deprecated.